test/config-next/ocsp-responder.json

```diff
@@ -65,5 +65,8 @@
 	"opentelemetry": {
 		"endpoint": "bjaeger:4317",
 		"sampleratio": 1
+	},
+	"openTelemetryHttpConfig": {
```

Comment:
Doesn't need to be fixed in this PR, but it looks like the casing for `opentelemetry` and are off. You've got the casing right in your new fields though.

Reply:
might as well fix it while I'm in here